src/main/java/spireMapOverhaul/zones/gremlinTown/powers/CriticalPower.java

```diff
@@ -0,0 +1,49 @@
+
+package spireMapOverhaul.zones.gremlinTown.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+import spireMapOverhaul.zones.gremlinTown.GremlinTown;
+
+import static spireMapOverhaul.util.Wiz.atb;
+
+public class CriticalPower extends AbstractSMOPower {
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID("Critical");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public CriticalPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, GremlinTown.ID, PowerType.BUFF,false, owner, amount);
+        priority = 7;
+    }
+
+    @Override
+    public void stackPower(int stackAmount) {
```

Comment:
You're missing some of the standard logic with this, not specifically needed, I suppose